README.md

```diff
@@ -18,7 +18,7 @@ You should have all of the following installed before installing GridKit™
 	- [Suitesparse](https://github.com/DrTimothyAldenDavis/SuiteSparse) >= 5.x (optional)
 	- [Ipopt](https://github.com/coin-or/Ipopt) >= 3.x (optional)
     - [Enzyme](https://github.com/EnzymeAD/Enzyme) >=0.0.131 (optional)
-        - [LLVM](https://github.com/llvm/llvm-project) >= 18.x 
+        - [LLVM](https://github.com/llvm/llvm-project) 16.x 
 - [CMake](https://cmake.org/) >= 3.12
```

Comment:
I would mirror what Enzyme team specifies for the dependency here:
```suggestion
        - [LLVM](https://github.com/llvm/llvm-project) >= 15.x 
- [CMake](https://cmake.org/) >= 3.12
```
We could add a note we are testing with v16.